contracts/casino/include/casino/casino.hpp

```diff
@@ -91,6 +134,7 @@ class [[eosio::contract("casino")]] casino: public eosio::contract {
                 row.quantity += quantity;
             });
         }
+        gstate.game_profits_sum += quantity * get_profit_margin(game_id) / 100;
```

Comment:
move percent to constants

Reply:
why?

Reply:
because of it's not good way to using numeric constants directly in code

Reply:
that's true, but in this case `const int percent_100 = 100;` won't do any good because it doesn't improve readability